apps/hiring-kanban/src/components/CardForms/CardHistory.tsx

```diff
@@ -172,6 +200,34 @@ export function CardHistory() {
             </h1>
           </div>
           {history && <Table columns={cols} data={data} />}
+
+          <Modal isOpen={!!showPreviewModal} handleClose={handleClose}>
+            {typeof showPreviewModal === 'number' &&
+              history?.[showPreviewModal] && (
+                <>
+                  <CardView card={history[showPreviewModal]!} />
+                  <button
+                    type="button"
+                    title="Previous"
+                    onClick={() => setShowPreviewModal(showPreviewModal + 1)}>
+                    <ChevronDoubleLeftIcon
+                      className="absolute -mt-60 ml-10  h-8 w-8 text-stone-400 hover:text-indigo-700"
+                      aria-hidden="true"
+                    />
+                  </button>
+                  <button
+                    type="button"
+                    title="Next"
+                    onClick={() => setShowPreviewModal(showPreviewModal - 1)}>
```

Comment:
you will need some validation I think, to stop people clicking outside the range of history items. Try adding a 'hasNextItem' function and disable the next button if its false. then disable the previous button if the current index is 0